yt_dlp/extractor/youtube.py

```diff
@@ -1462,6 +1489,226 @@ def _extract_yt_initial_variable(self, webpage, regex, video_id, name):
             (r'%s\s*%s' % (regex, self._YT_INITIAL_BOUNDARY_RE),
              regex), webpage, name, default='{}'), video_id, fatal=False)
 
+    @staticmethod
+    def _text_entries(runs):
+        for run in runs:
+            if not isinstance(run, dict):
+                continue
+            sub_text = try_get(run, lambda x: x['text'], compat_str)
+            if sub_text:
+                yield sub_text
+
+    def _join_text_entries(self, runs):
+        text = ''.join(t for t in self._text_entries(runs))
+        if text:
+            return text
```

Comment:
seems unnecessary to be 2 seperate functions since we only ever have direct use for `_join_text_entries`